src/apps/Registration.js

```diff
@@ -56,15 +55,16 @@ const RegistrationComponent = (props) => {
                     firebase
                         .auth()
                         .createUserWithEmailAndPassword(email, pass)
-                        .then((user) => {
-                            writeData();
-                            const docRef = firebase
-                                .firestore()
+                        .then(({ user }) => {
+                            firebase.firestore()
                                 .collection("Users")
-                                .doc();
-                            docRef.set({
-                                Name: document.getElementById("name").value,
-                                Email: document.getElementById("email").value,
+                                .doc(user.uid)
+                                .set({
+                                name, email,
```

Comment:
should this be done?
`name.toLowerCase()`

Reply:
Nope, we want to preserve the casing that the user enters